### ISSUE TITLE
Strip Alpha channel from thumbs

### ISSUE THREAD

**Issue by kokoko3k** [libretro-thumbnails/libretro-thumbnails#137]
I've seen that Named_Boxarts contains pngs with an useless alpha channel.
Stripping it provides a good reduction in image size.
eg:
Commodore - Amiga/Named_Boxarts/1869_v1.0_AGA.png : 631.3KB with alpha channel
Commodore - Amiga/Named_Boxarts/1869_v1.0_AGA.png : 561.5KB with alpha channel stripped

(The stripped file has been saved via gimp with png compression set to 9, which I think is the default used compression, since saving it with the Gimp WITH alpha channel and compression=9 leads to very similar file size.)

**Comment by RobLoach** [comment]
Sounds fine with me. We'll want to test to make sure they still render correctly in RetroArch. imagemagik or similar may help you out.

**Comment by kokoko3k** [comment]
> Sounds fine with me. We'll want to test to make sure they still render correctly in RetroArch. imagemagik or similar may help you out.

Yeah thanks, i already used pngquant to 8bpc for all my collection :)
